frontend/my-app/src/components/CreateMeeting.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+
+export default function CreateMeeting() {
+    const [meetingInfo, setInfo] = useState({
+        code: "",
+        duration: "",
+        type: ""
+    })
+    const [isPending, setPending] = useState(false);
+    function submit(e) {
+        e.preventDefault();
+        setPending(true);
+        const data = {
+            code: meetingInfo.code,
+            duration: meetingInfo.duration,
+            type: meetingInfo.type
+        }
+        console.log(data);
+        fetch('http://localhost:5000/meeting', {
```

Comment:
Organization wise, I would create a new file containing the URLs for dev and prod and call it in your frontend code

fetch(`${URL}/members`)

Where URL is a variable in another file which can take on different url's based on production or development. As  result, all the API prefixes we call are standardized and can easily be toggled.

Does that make sense?

Does that make sense?

Reply:
Yep makes sense that sounds good